spec/models/comment_spec.rb

```diff
@@ -84,6 +84,22 @@
         expect(subject).not_to be_valid
       end
     end
+
+    describe "#mention_total" do
+      it "is valid with less than six mentions in markdown" do
+        subject.commentable_type = "Article"
+
+        subject.body_markdown = "hi @#{user.username}! " * 6
+        expect(subject).to be_valid
+      end
+
+      it "is invalid with more than six mentions in markdown" do
+        subject.commentable_type = "Article"
+
+        subject.body_markdown = "hi @#{user.username}! " * 7
+        expect(subject).not_to be_valid
```

Comment:
Sometimes it's nice to explicitly verify the error message with `not_to be_valid` to ensure it wasn't something else that caused the record to become invalid (note the case here, but maybe changing from `* 6` to `* 7` could have made the record invalid because `body_markdown` is too long or something):

```ruby
expect(subject.errors[:base])
  .to include("You cannot mention more than 6 users in a comment!")
```

Reply:
Great idea! 👍🏽 Makes for a much more robust test, too.